build-system/tasks/runtime-test.js

```diff
@@ -305,6 +305,7 @@ function unitTestsToRun() {
   const filesChanged = gitDiffNameOnlyMaster();
   const testsToRun = [];
   const srcFiles = [];
+  testsToRun.push('test/simple-test.js');
```

Comment:
nit: You could move this to the initialization in line 306, and add a comment pointing to this PR.